src/esm_runscripts/namelists.py

```diff
@@ -265,8 +267,8 @@ def echam_transient_forcing(config):
                 import pandas as pd
 
                 try:
-                    forcing_table = pd.read_csv(
-                        config["echam"]["transient_forcing_table"]
+                    forcing_table = pd.open_csv(
+                        config["echam"]["transient_forcing_table"], sep=";", index_col=0
```

Comment:
My only comment is here: `open_csv` didn't seem to exist and we changed to `read_csv` so, why is `open_csv` back? Is this once again one of this times where GitHub is performing poorly on showing the changes of a merge?

For the rest of the changes, I did not spot anything that affects to any other ESM-Tools functionalities.

Reply:
That's GitHub being messy again...

Reply:
Check again please, I just fixed it...

Reply:
Looks good to me, from my side, feel free to merge whenever you consider it's ready.